src/main/java/mate/academy/dao/CinemaHallDao.java

```diff
@@ -0,0 +1,16 @@
+package mate.academy.dao;
+
+import java.util.List;
+import java.util.Optional;
+import mate.academy.lib.Dao;
+import mate.academy.model.CinemaHall;
+
+@Dao
```

Comment:
annotations shouldn't be in interfaces. only in implementations. fix in all places